src/App.tsx

```diff
@@ -70,22 +85,23 @@ const App: React.FC = () => (
     <GlobalStyles />
     <Router basename={process.env.BASE_URL}>
       <Layout>
-        <Switch>
-          <PrivateRoute path="/orders" exact component={Orders} />
-          <Route path="/trade/:sell-:buy" component={Trade} />
-          <PrivateRoute path="/liquidity" exact component={Strategies} />
-          <PrivateRoute path="/wallet" exact component={Wallet} />
-          <PrivateRoute path="/orders" exact component={Orders} />
-          <Route path="/about" exact component={About} />
-          <Route path="/source-code" exact component={SourceCode} />
-          <Route path="/connect-wallet" exact component={ConnectWallet} />
-          <Redirect from="/" to="/trade/DAI-USDC?sell=0&buy=0" />
-          <Route component={NotFound} />
-        </Switch>
+        <React.Suspense fallback={null}>
```

Comment:
There generally won't be an error as it loads what is always available -- chunk files
Plus that's how Suspense works -- it catches a thrown Promise and shows fallback, when promise resolves -- shows actual component

But it may be a good idea to add a general ErrorBoundary above Suspense
Sometime later